lib/hippie/client.js

```diff
@@ -13,6 +13,7 @@ var http = require('http');
 var serializers = require('./serializers');
 var parsers = require('./parsers');
 var expect = require('./expect');
+var Promise = require('es6-promise').Promise;
```

Comment:
people may want to switch the `Promise` library to whatever they use in their app. I'd suggest we make this configurable somehow